code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -613,9 +613,9 @@
 
 /datum/reagent/toxin/amanitin/on_mob_delete(mob/living/M)
 	var/toxdamage = current_cycle*3*REM
-	M.adjustToxLoss(toxdamage)
 	if(M)
-		add_logs(M, get_turf(M), "has taken [toxdamage] toxin damage from amanitin toxin")
+		M.adjustToxLoss(toxdamage)
+		M.log_message("has taken [toxdamage] toxin damage from amanitin toxin", INDIVIDUAL_ATTACK_LOG)
```

Comment:
Hmm, would probably be better, since the logs will be stored on the ckey aswell, so you won't miss any logging.